js/state/frameStateUtil.js

```diff
@@ -717,6 +722,38 @@ const setTabHoverState = (state, frameKey, hoverState, enablePreviewMode) => {
   return state
 }
 
+const getTabPageIndex = (state) => {
+  const tabPageIndex = state.getIn(['ui', 'tabs', 'tabPageIndex'], 0)
+  const previewTabPageIndex = state.getIn(['ui', 'tabs', 'previewTabPageIndex'])
+
+  return previewTabPageIndex != null ? previewTabPageIndex : tabPageIndex
```

Comment:
can we simplify this to `return state.getIn(['ui', 'tabs', 'previewTabPageIndex']) || tabPageIndex`